New package: BOSS v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: BOSS
- Repository: https://github.com/soldasim/BOSS.jl
- Created by: @soldasim
- Version: v0.1.0
- Commit: 0fb53c7ffaef80a7f7aa9d390a4ef54edf2ff7ea
- Reviewed by: @soldasim
- Reference: https://github.com/soldasim/BOSS.jl/issues/37#issuecomment-2233069936
- Description: BOSS (Bayesian Optimization with Semiparametric Surrogate)
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598480b37e941b5a47387fc255c9318702915b8a79ab29b6f2c95238526f6bc32418fccc5a14794246353571176d94ef0e6ef4605d1d7a27850393b145db85a4de4b433689d4ad9db0dda35c6afe2a5d545f95afe0f64a1a0d2f338ff71786614e3dd8917f127bbc12d5f050445f6d6842a -->

goerz commented:
I think the name is fine, in the category of "less systematic names". In my book, pronounceable acronyms make perfectly fine package names, and all-caps are okay in that situation (we'd accept `Boss`, so we should also accept `BOSS`).

However, it will need a manual merge, so pint the `#pkg-registration` channel on Slack after the end of the tree-day waiting period.

[noblock]

goerz commented:
[noblock] The compat issues have to be fixed before this can be merged, of course.

JuliaRegistrator committed:
New package: BOSS v0.1.0

UUID: 15052f3f-a82a-4ca3-bbc4-f478a3bd1ae2
Repo: https://github.com/soldasim/BOSS.jl.git
Tree: 0473af05d19187080f0e0fc9a582290df265c80a

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
